check/check.go

```diff
@@ -138,12 +123,23 @@ type Group struct {
 func (c *Check) Run(definedConstraints map[string][]string) {
 	// If check type is skip, force result to INFO
 	if c.Type == "skip" {
+		c.Reason = "Test marked as skip"
 		c.State = INFO
 		return
 	}
 
 	//If check type is manual or the check is not scored, force result to WARN
-	if c.Type == "manual" || !c.Scored {
+	if c.Type == "manual" {
+		c.Reason = "Test marked as a manual test"
+		c.State = WARN
+		return
+	}
+
+	// Since this is an Scored check
+	// without tests return a 'WARN' to alert
+	// the user that this check needs attention
+	if c.Scored && len(strings.TrimSpace(c.Type)) == 0 && c.Tests == nil {
```

Comment:
Don't we want to warn in this case even if it's not Scored?

Reply:
I would be OK with adding this as a separate issue so that we can get this one merged quickly

Reply:
Oh I see, yes I just thought that if it's not scored then it would still be warn and not fail but there will be a difference in the warn reason, I guess that the only changes between a scored and not scored tests will need to be in failed tests status, WARN instead of FAIL yes? 
Or maybe we want any other differences? 
In any case I think that open a different PR will be best so we could marge this one, and then add or change small stuff in different PRs :)

Reply:
Currently removed the c.Scored parameter and not scored and scored tests will behave the same in this case.
I opened a PR https://github.com/aquasecurity/bench-common/issues/91 about the subject so we could understand the way we want to deal with not scored and scored tests